agents/plugins/mk_logwatch.py

```diff
@@ -1296,19 +1298,27 @@ def main(argv=None):  # pylint: disable=too-many-branches
             filtered_log_lines,
         )
 
-    process_batches(
-        [ensure_str(l) for l in output],
-        batch_id,
-        REMOTE,
-        global_options.retention_period,
-        now,
-    )
+    try:
+        process_batches(
+            [ensure_str(l) for l in output],
+            batch_id,
+            REMOTE,
+            global_options.retention_period,
+            now,
+        )
+    except Exception as exc:
```

Comment:
Do you know what exceptions occur here?

Reply:
Yes. Exceptions like:
- 'ascii' codec can't decode byte 0xef in position 77: ordinal not in range(128)
- 'ascii' codec can't decode str '\ufffd' in position 127: ordinal not in range(128)

Reply:
@andiumbreit is currently fixing bugs in mklogwatch related to py2/3. He might already found your other issues

Reply:
Cool. So I will wait for a fix and related werk which explains those fixes for the customers.

Thank you @kain88-de, @andiumbreit